DigitalLearningSolutions.Data/Services/CentresService.cs

```diff
@@ -48,5 +50,16 @@ FROM Centres
 
             return name;
         }
+
+        public IEnumerable<(int, string)> GetActiveCentres()

```

Comment:
Ah, we should probably add a test for the new data methods and as per my comment in the hee-internal channel, move them to a new "DataServices" folder (which will effectively be repositories)

Reply:
I'm having trouble thinking of the sort of tests we can add for GetActiveCentres. I guess that it returns at least one centre, but we can't even check that they're all active without extending the method.